### ISSUE TITLE
Remove subspace UI support

### ISSUE THREAD

**Issue by alfred-landrum** [brimdata/brim#1397]
A "subspaces" is a space created by issuing queries against specific zng files in a separate archive backed space. This feature is a leftover of some earlier plans that we are no longer pursuing. We should remove the lingering UI support, which we never exposed outside of internal demos. Once the UI work is done, we'll remove the api, zqd, zapi, and zar code as well, I'll file a separate issue for that.

**Comment by alfred-landrum** [comment]
#1473 includes related Brim side code removal.
